src/ontology/cl-edit.owl

```diff
@@ -24008,33 +24063,36 @@ TA: Transit Amplifying Cells") foaf:depiction obo:CL_0009080 <http://purl.obolib
 EquivalentClasses(obo:CL_0009080 ObjectIntersectionOf(obo:CL_0019032 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002108)))
 SubClassOf(obo:CL_0009080 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001902))
 
-# Class: obo:CL_0009081 (specified double negative thymocyte (Homo sapiens))
+# Class: obo:CL_0009081 (specified double negative thymocyte (Hsap))
```

Comment:
Sigh… You gotta love the consistency of AI tools. For all the terms above whose label have been changed, the agent didn’t bother to update the comment to reflect the new label, and here all of a sudden it decides that it won’t do, and that the comments must be updated.

I guess deterministic tools are just boring.